repose-aggregator/core/core-lib/pom.xml

```diff
@@ -137,6 +138,18 @@
 
         <dependency>
             <groupId>org.openrepose</groupId>
+            <artifactId>atom-feed-service-api</artifactId>
+            <version>${project.version}</version>
+        </dependency>
+
+        <dependency>
```

Comment:
It's also done for the httpclient service. How would the api and implementation classes be available to use at runtime if they weren't added to some JAR? For anything that will use this service, only the api will be needed, but here we just want to make the service usable for any installation of Repose.